app/modules/asset_groups/models.py

```diff
@@ -495,6 +495,202 @@ def get_all_jobs_debug(cls, verbose):
             jobs.extend(asset_group_sighting.get_jobs_debug(verbose))
         return jobs
 
+    def get_pipeline_status(self):
+        db.session.refresh(self)
+        status = {
+            'preparation': self._get_pipeline_status_preparation(),
+            'detection': self._get_pipeline_status_detection(),
+            'identification': self._get_pipeline_status_identification(),
+            'now': datetime.utcnow().isoformat(),
+            'stage': self.stage,
+            'migrated': False,  # always false, but just for consistency with sighting
+            'summary': {
+                'progress': None,
+            },
+        }
+        status['summary']['complete'] = (
```

Comment:
The AGS is only 'complete' ie finished once the user has done the commit so I'm not sure this is accurate. 
But I'm also not sure what is better, shage== finished?

Reply:
the shortest answer i can give here is "i think this is a _different_ complete".   :/     in particular, the **summary** definition of "complete" (even for this usage) is likely to change a bit as we play with this.  if its even something we can definitively say at all!